Source/Building_BedMedPod.cs

```diff
@@ -347,11 +347,11 @@ private void DiagnosePatient(Pawn patientPawn)
 
             foreach (Hediff currentArtificialPartHediff in artificialPartHediffs)
             {
-                childPartsToSkip.AddRange(GetBodyPartDescendants(currentArtificialPartHediff.part));
+                childPartsToSkip.AddRange(GetBodyPartDescendants(currentArtificialPartHediff.Part));
```

Comment:
Ah, I see now why you changed `part` to `Part` - the latter is public and has null handling.